### PR TITLE
add docs for creating attestation secret.

### PR DESCRIPTION
**Pull request by tejal29**

### CODE REVIEW

File: tutorial.md [hunk 1]
```diff
@@ -35,8 +35,63 @@ spec:
...
+sub   rsa3072 2018-06-14 [E]
+
+gpg --export-secret-keys --armor C8C9D53FAE035A650B6B12D3BFF4AC9F1EED759C > gpg.priv
+```
+Now create a secret using the exported public and private keys
+```shell
+kubectl create secret generic my-attestator --from-file=public=gpg.pub --from-file=private=gpg.priv
+```
+Finally create an attestation authority
+1. Grab the base encoded value of public key for the secret you just created.
```
[the remark this answers is not in the document]
Comment (tstromberg) [reply tutorial.md]: This step is weird: we've already exported the public key with --armor, which outputs base64 (well, technically Radix-64) output. Why the need to run it through base64 again?

**Comment by tejal29** [comment]
From bug bash
- [ ] In creating Attestation Authority step, the code snippet contains the output of the command. Remove it or split into multiple shells.
- [ ] Remove `PROJECT_ID=<>` from the steps for creating  Attestation Authority
- [ ] The output listed

File: tutorial.md [hunk 2]
```diff
@@ -35,8 +35,62 @@ spec:
       - providers/goog-vulnz/notes/CVE-2017-1000081
 EOF
 ```
+### 2. Setting up an AttestationAuthority
```
[the remark this answers is not in the document]
Comment (tejal29) [reply tutorial.md]: yes we would want users to go through the exercise of setting up AA. 
Its not required for tutorial,  but then admitted images would not be attested. In production, you would want to set up AA.

File: tutorial.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (tejal29) [reply tutorial.md]: The output of --armor is PEM format which will result in invalid YAML value. Hence base64 encoded value is required.

File: tutorial.md [hunk 3]
```diff
@@ -35,8 +35,63 @@ spec:
...
+spec:
+    noteReference: v1alpha1/projects/$PROJECT
+    privateKeySecretName: my-attestator
+    publicKeyData: $PUBLIC_KEY
+EOF
+```
+
+The `AttestationAuthority` should be configured:
+```shell
+attestationauthority.kritis.grafeas.io/my-attestator created
```
[the remark this answers is not in the document]
Comment (tejal29) [reply tutorial.md]: i think the creating image security policy shows it. I wanted to keep both steps in sync so added this line.

File: tutorial.md [hunk 4]
```diff
@@ -35,8 +35,63 @@ spec:
...
+```
+This `AttestationAuthority` will create Attestation Note in project specified in `$PROJECT` variable and attest valid images using the secret `my-attestator` which we created.
```
[the remark this answers is not in the document]
Comment (tejal29) [reply tutorial.md]: hmm i can add a link to the term in here.

File: tutorial.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (tejal29) [reply tutorial.md]: hmm maybe you deleted it recently.

File: tutorial.md [hunk 5]
```diff
@@ -35,8 +35,62 @@ spec:
...
+Kritis relies on user defined AttestationAuthorities to attest images admitted. Attested images will be always admitted in future.
+To create a gpg public, private key pair run,
+Note: Please create a key with Empty Passphase. We are working on adding support for [passphrase](https://github.com/grafeas/kritis/issues/186)
+```shell
+gpg --quick-generate-key --yes my.attestator@example.com
+
+gpg --armor --export my.attestator@example.com > gpg.pub
+
+gpg --list-keys my.attestator@example.com
+pub   rsa3072 2018-06-14 [SC] [expires: 2020-06-13]
```
[the remark this answers is not in the document]
Comment (tejal29) [reply tutorial.md]: done. Verified i get the same private key when using the id or user name.

File: tutorial.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (tejal29) [reply tutorial.md]: done

File: tutorial.md [hunk 6]
```diff
@@ -35,8 +35,62 @@ spec:
       - providers/goog-vulnz/notes/CVE-2017-1000081
 EOF
 ```
+### 2. Setting up an AttestationAuthority
+Kritis relies on user defined AttestationAuthorities to attest images admitted. Attested images will be always admitted in future.
+To create a gpg public, private key pair run,
```
[the remark this answers is not in the document]
Comment (tejal29) [reply tutorial.md]: done

**Pull request merged**